loader.lua

```diff
@@ -1,6 +1,33 @@
-dofile('data/map_generator/core/func.lua')
-dofile('data/map_generator/core/map.lua')
-dofile('data/map_generator/core/draw.lua')
+------------------------------
+-- Loader config
+------------------------------
+local mapGeneratorDir = "data/map_generator-master/"
+
+------------------------------
+-- Core files loader
+------------------------------
+print('Loading map generator core files...')
```

Comment:
map_lib_cfg.debugOutput - this value in config file sets console output to true or false